### ISSUE TITLE
How to run tasks parallel to the Slack event loop with a request URL

### ISSUE THREAD

**Issue by ipelkonen** [slackapi/bolt-python#799]
Hi,
I would like to run some work parallel to the bolt-python Slack event loop, to be able to send Slack messages from time to time from this code, using the same data structures in both event handling and when sending those messages from outside the event loop.

The process is as follows:

1. Wait for external trigger
2. Send message
3. Wait for reaction
4. Process reaction

There can be different messages waiting for reactions from many (and different) users simultaneously, and a message shouldn't be necessary at program start before reactions can be monitored, so these should work in parallel. It can also take some time for users to react.

I saw that in issue #592 a similar thing is already done for socket mode apps but I'm using a request URL so I hope it is possible to implement this with that as well. I just don't seem to get it right by myself so I hope you can help.

Thanks in advance!

**Comment by ipelkonen** [comment]
Hi @seratch, thanks for the reply! I appreciate the work you do and understand that the resources are limited. I'm not deep enough into asynchronous Python programming to continue on my own so I'm switching the implementation language to Javascript. Luckily Python is not must for this project.

Thanks for your time!
